tools/utils/Utils/Settings.StyleCop

```diff
@@ -0,0 +1,37 @@
+<StyleCopSettings Version="105">
+    <GlobalSettings>
+        <CollectionProperty Name="RecognizedWords">
+            <Value>Pii</Value>
+            <Value>Etw</Value>
+            <Value>osgwiki</Value>
+            <Value>appx</Value>
+            <Value>appxbundle</Value>
+            <Value>blockmap</Value>
+            <Value>cmd</Value>
+            <Value>dapx</Value>
+            <Value>msix</Value>
+            <Value>msixbundle</Value>
+        </CollectionProperty>
+    </GlobalSettings>
+    <Analyzers>
+        <Analyzer AnalyzerId="StyleCop.CSharp.DocumentationRules">
+            <Rules>
+                <Rule Name="ElementsMustBeDocumented">
+                    <RuleSettings>
+                        <BooleanProperty Name="Enabled">False</BooleanProperty>
+                    </RuleSettings>
+                </Rule>
+            </Rules>
+            <AnalyzerSettings />
+        </Analyzer>
+        <Analyzer AnalyzerId="Microsoft.StyleCop.CSharp.NamingRules">

```

Comment:
>icrosoft.StyleCop. [](start = 31, length = 18)

does this work with public stylecop? does this need to be updated to public stylecop?

Reply:
I believe this is already public StyleCop.